src/lib/getModelData.ts

```diff
@@ -90,7 +91,7 @@ export async function getModelData(url: string, options: Partial<Options> = {})
       if (node instanceof Mesh) {
         bodies++
         if (node.material) {
-          materials++
+          materials.add((node.material as Material).name)
```

Comment:
The idea behind this set is to avoid counting multiple times the reused materials for different objects